### PR TITLE
Feature/solvers

### PR DESCRIPTION
**Pull request by rolfvdhulst**
Some minor changes to make the infrastructure more friendly/standardized for the use of solvers/ other classes.

### CODE REVIEW

File: include/roboteam_world/filters/BallObservation.h [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+// Created by rolf on 21-01-20.
+//
+
+#ifndef RTT_BALLOBSERVATION_H
+#define RTT_BALLOBSERVATION_H
+#include <roboteam_proto/messages_robocup_ssl_detection.pb.h>
+/**
+ * A struct to keep Ball Data and time as one observation.
+ */
+struct BallObservation {
```
Comment (Yuhanun) [inline include/roboteam_world/filters/BallObservation.h]: no namespace?

File: include/roboteam_world/filters/RobotObservation.h [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+// Created by rolf on 21-01-20.
+//
+
+#ifndef RTT_ROBOTOBSERVATION_H
+#define RTT_ROBOTOBSERVATION_H
+#include <roboteam_proto/messages_robocup_ssl_detection.pb.h>
+/**
+ * A struct to keep robotData and time as one observation. We can pass this around to various classes easily.
+ */
+struct RobotObservation {
```
Comment (Yuhanun) [inline include/roboteam_world/filters/RobotObservation.h]: namespace?

**Review by Yuhanun — APPROVED** [review]

File: include/roboteam_world/filters/BallObservation.h (hunk 1 above)
Comment (rolfvdhulst) [reply include/roboteam_world/filters/BallObservation.h]: Nothing in the repo is namespaced right now.

Comment (Yuhanun) [reply include/roboteam_world/filters/BallObservation.h:11]: Oh okay, approved then

**Pull request merged**